packages/teleterm/webpack.renderer.dev.config.js

```diff
@@ -12,7 +12,7 @@ devCfg.devServer = {
     directory: path.join(__dirname, 'build/app/dist/renderer'),
     serveIndex: false,
   },
-  allowedHosts: 'auto',
+  allowedHosts: 'all',
```

Comment:
This was helpful for testing webauthn stuff, since webauthn doesn't seem to like localhost and so I needed to use proxy.0.0.0.0.nip.io as the configured domain instead.